src/common/config.h

```diff
@@ -124,6 +124,7 @@ struct md_config_t {
   int parse_buffer(ConfigValues& values, const ConfigTracker& tracker,
 		   const char* buf, size_t len,
 		   std::ostream *warnings);
+  void update_legacy_vals(ConfigValues& values);
```

Comment:
nit: a typo in the commit msg (`popolated`).

Reply:
fixed.